src/Platform/Hosted/Components/ConnectExplorer.tsx

```diff
@@ -30,6 +30,28 @@ export const fetchEncryptedToken = async (connectionString: string): Promise<str
   return decodeURIComponent(result.readWrite || result.read);
 };
 
+export const isAccountRestrictedForConnectionStringLogin = async (connectionString: string): Promise<boolean> => {
+  const headers = new Headers();
+  headers.append(HttpHeaders.connectionString, connectionString);
+  const url = configContext.BACKEND_ENDPOINT + "/api/accountrestrictions/checkconnectionstringlogin";
+  const response = await fetch(url, { headers, method: "POST" });
+  if (!response.ok) {
+    throw response;
+  }
+
+  return (await response.text()) === "True";
+};
+
+export const hideErrorDetails = () => {
+  window.document.getElementById("errorDetails").innerText = "";
```

Comment:
You don't want to edit the DOM directly like this. If logic is self contained within a component, use state (like line 62) is the way to go. There are tons of examples throughout this repo's *.tsx files showing various React-native ways to alter the UI.